eZ/Bundle/EzPublishCoreBundle/Command/CleanupVersionsCommand.php

```diff
@@ -40,24 +40,9 @@ class CleanupVersionsCommand extends Command
     private $repository;
 
     /**
-     * @var \eZ\Publish\API\Repository\UserService
+     * @var \eZ\Publish\Core\MVC\ConfigResolverInterface
      */
-    private $userService;
-
-    /**
-     * @var \eZ\Publish\API\Repository\ContentService
-     */
-    private $contentService;
-
-    /**
-     * @var \eZ\Publish\API\Repository\PermissionResolver
-     */
-    private $permissionResolver;
-
-    /**
-     * @var \eZ\Bundle\EzPublishCoreBundle\ApiLoader\RepositoryConfigurationProvider
-     */
-    private $repositoryConfigurationProvider;
+    private $resolver;
```

Comment:
We usually name it `$configResolver` and it makes sense to me, because resolver is too vague (can apply to many different things).

Reply:
Fixed in a743866